core/bessctl.cc

```diff
@@ -531,17 +562,72 @@ class BESSControlImpl final : public BESSControl::Service {
 
   Status ResumeAll(ServerContext*, const EmptyRequest*,
                    EmptyResponse*) override {
-    LOG(INFO) << "*** Resuming ***";
     if (!is_any_worker_running()) {
       attach_orphans();
     }
+
+    ResumeHookUser<Worker>* global_hooks =
+        bess::ResumeHookUser<Worker>::get_user_hooks(nullptr);
+
+    // Enable metadata offset computation by default
```

Comment:
I wish there were a good way to get this to happen at constructor time, when we build the metadata resume hook... not sure if there is though, we might get into constructor ordering issues.

Reply:
Yeah, I thought about putting this logic in the [Pipeline constructor](https://github.com/NetSys/bess/blob/master/core/metadata.h#L148), but it's not clear how to handle re-adding the hook after a `daemon reset`.